code/modules/vore/eating/belly_obj_vr.dm

```diff
@@ -441,6 +441,9 @@
 /obj/belly/AllowDrop()
 	return TRUE
 
+/obj/belly/onDropInto(var/atom/movable/AM)
+	return src
```

Comment:
Looks like onDropInto is a sort of half-kinda-sorta partial implementation of drop_location from Bay or something. It appears to only be used two places. Maybe just remove it and replace with drop_location?

Reply:
This would need changes to the dropInto proc as it happens to be one of those things critically using the onDropInto

Reply:
Actually I kinda have no idea how to do it. The things would have to be dropping into the prey mob's drop_location, and beyond remove_from_mob's dropInto there isn't much room for the prey mob. Possibly change the dropInto's "destination.onDropInto(src)" part into like "loc.drop_location" of the item?

Reply:
I had a really long post typed up, but then started looking at the /tg/ implementation that I was going to suggest using, and it's a lot of typing. I'll just do it after this PR I guess.

Reply:
Alright should work.